ColabTool/Shared/Post/index.jsx

```diff
@@ -8,10 +8,10 @@ import PostFooter from '../PostFooter';
 import PostHeader from '../PostHeader';
 import RetweetPanel from '../RetweetPanel';
 
-const renderRetweetComment = ({retweetComment, links}) => (
+const renderRetweetComment = ({retweetComment, retweetCommentLinks}) => (
   <div className={style.comment}>
     <LinkifiedText
-      links={links}
+      links={retweetCommentLinks}
```

Comment:
In this file, replacing `links` (which corresponds to retweet text links) with `retweetCommentLinks` which is specifically for the comment. 

Also removing references to `links` since we don't need it here.